### PR TITLE
Bugfix for AR events.py

### PR DESCRIPTION
**Pull request by a-cat-named-snowball**
Modified AR events.py to scrape bill id's from the agenda pdf.

### CODE REVIEW

**Review by chrisyamas — CHANGES_REQUESTED** [review]
Great fix, the only issue found in testing it on the [page](https://www.arkleg.state.ar.us/Calendars/Meetings?tbType=&meetingStartDate=2023-03-02&meetingEndDate=2023-05-31) is that it's not scraping the agenda for the "JUDICIARY COMMITTEE - SENATE" meeting on Monday, March 6th.

That's the only one that doesn't get triggered. Can you investigate why and write a solution so we're getting bills from all agendas?

File: scrapers/ar/events.py [hunk 1]
```diff
@@ -110,13 +111,19 @@ def scrape(self, start=None, end=None):
...
+
+        # Remove duplicates
+        matches = list(set(matches))
 
-            for row in page.xpath("//div[contains(@class,'measureTitle')]/a/text()"):
-                agenda.add_bill(row.strip())
+        # Yield results, ignoring an a duplicate part of the regex
+        for m, _ in matches:
+            # Add in space between letters and numbers
+            yield m[:2] + " " + m[2:]
```
Comment (johnseekins) [inline scrapers/ar/events.py:129]: Maybe `f"{m[:2]} {m[2:]}"` instead?

Comment (a-cat-named-snowball) [reply scrapers/ar/events.py]: Yes that would be easier to read - fixed with https://github.com/openstates/openstates-scrapers/pull/4518/commits/f82e7b161ceff5f7ec1501a267fac7ca0df5d36b Thanks :smile:

**Comment by a-cat-named-snowball** [comment]
Data for "JUDICIARY COMMITTEE - SENATE" is being scraped correctly now, without changes to relevant code. The agenda pdf may have been in the process of being updated when the scraper was ran.

**Comment by chrisyamas** [comment]
> Data for "JUDICIARY COMMITTEE - SENATE" is being scraped correctly now, without changes to relevant code. The agenda pdf may have been in the process of being updated when the scraper was ran.

@a-cat-named-snowball I just ran it again, and this is still the JSON data I get. The `"agenda"` list is empty, while it should contain a collection that includes Associated Bills with a `"related_entities"` key that contains the actual bill names and such:

`{"name": "JUDICIARY COMMITTEE - SENATE", "all_day": false, "start_date": "2023-03-06T16:00:00+00:00", "end_date": "", "status": "confirmed", "classification": "event", "description": "", "upstream_id": "", "location": {"name": "Room 171, Arkansas State Capitol, 500 Woodlane St, Little Rock, AR 72201", "note": "", "coordinates": {"latitude": "34.74640441284063", "longitude": "-92.2895327976071"}}, "media": [], "documents": [], "links": [], "participants": [{"name": "JUDICIARY COMMITTEE - SENATE", "entity_type": "organization", "note": "host", "organization_id": "~{\"name\": \"JUDICIARY COMMITTEE - SENATE\"}"}], "agenda": [], "sources": [{"url": "https://www.arkleg.state.ar.us/Calendars/Meetings", "note": ""}], "extras": {}, "_id": "810f8d9c-b94c-11ed-bb71-0242ac120002"}`

Here also is a section of console output that shows the `Agenda` class scraper called on the urls for the 2023-03-02 HOUSE MANAGEMENT meeting, *no execution of an `Agenda` scrape for 2023-03-06 JUDICIARY COMMITTEE - SENATE meeting, but then two more calls to `Agenda` for the following two items that have them. Does your console output match or differ?

**Comment by a-cat-named-snowball** [comment]
It does differ, here's the data I'm getting:

`{"name": "JUDICIARY COMMITTEE - SENATE", "all_day": false, "start_date": "2023-03-06T16:00:00+00:00", "end_date": "", "status": "confirmed", "classification": "event", "description": "", "upstream_id": "", "location": {"name": "Room 171, Arkansas State Capitol, 500 Woodlane St, Little Rock, AR 72201", "note": "", "coordinates": {"latitude": "34.74640441284063", "longitude": "-92.2895327976071"}}, "media": [{"note": "Video of Hearing", "links": [{"url": "https://sg001-harmony.sliq.net/00284/Harmony/en/PowerBrowser/PowerBrowserV2/20230306/-1/27667?viewMode=1", "media_type": "text/html"}], "date": "", "classification": ""}], "documents": [{"note": "Agenda", "links": [{"url": "https://www.arkleg.state.ar.us/Calendars/Agenda?id=51502420&committee=420&agenda=25696&date=3%2F6%2F2023", "media_type": "application/pdf"}], "date": "", "classification": ""}], "links": [], "participants": [{"name": "JUDICIARY COMMITTEE - SENATE", "entity_type": "organization", "note": "host", "organization_id": "~{\"name\": \"JUDICIARY COMMITTEE - SENATE\"}"}], "agenda": [{"description": "Associated Bills", "classification": [], "related_entities": [{"name": "SB 347", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 347\"}"}, {"name": "SB 311", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 311\"}"}, {"name": "HB 1472", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1472\"}"}, {"name": "SB 283", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 283\"}"}, {"name": "SB 195", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 195\"}"}, {"name": "SB 182", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 182\"}"}, {"name": "HB 1394", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1394\"}"}, {"name": "SB 281", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 281\"}"}, {"name": "SB 2", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 2\"}"}, {"name": "SB 238", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 238\"}"}, {"name": "SB 342", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 342\"}"}, {"name": "SB 8", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 8\"}"}, {"name": "SB 337", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 337\"}"}, {"name": "HB 1455", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1455\"}"}, {"name": "SB 309", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 309\"}"}, {"name": "SB 266", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 266\"}"}, {"name": "HB 1426", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1426\"}"}, {"name": "SB 7", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 7\"}"}, {"name": "SB 40", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 40\"}"}, {"name": "HB 1427", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1427\"}"}, {"name": "HB 1431", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1431\"}"}, {"name": "HB 1474", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1474\"}"}, {"name": "HB 1493", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1493\"}"}, {"name": "SB 237", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 237\"}"}, {"name": "SB 96", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 96\"}"}, {"name": "SB 60", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 60\"}"}, {"name": "SB 270", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 270\"}"}, {"name": "SB 346", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 346\"}"}, {"name": "SB 312", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"SB 312\"}"}, {"name": "HB 1494", "entity_type": "bill", "note": "consideration", "bill_id": "~{\"identifier\": \"HB 1494\"}"}], "subjects": [], "media": [], "notes": [], "order": 0, "extras": {}}], "sources": [{"url": "https://www.arkleg.state.ar.us/Calendars/Meetings", "note": ""}], "extras": {}, "_id": "8b0e3a04-b94d-11ed-b79a-74e5f9d5f8cc"}`

Log near the relevant area:
```
18:00:46 INFO openstates: save event 2023-03-06 09:30:00-06:00 TRANSPORTATION, TECHNOLOGY & LEGISLATIVE AFFAIRS - SENATE as event_1068863c-b94e-11ed-a6a9-74e5f9d5f8cc.json
18:00:46 INFO ar.events.Agenda: fetching https://www.arkleg.state.ar.us/Calendars/Agenda?id=51502420&committee=420&agenda=25696&date=3%2F6%2F2023
18:00:46 INFO scrapelib: GET - 'https://www.arkleg.state.ar.us/Calendars/Agenda?id=51502420&committee=420&agenda=25696&date=3%2F6%2F2023'
18:00:47 INFO openstates: save event 2023-03-06 10:00:00-06:00 JUDICIARY COMMITTEE - SENATE as event_11113188-b94e-11ed-a6a9-74e5f9d5f8cc.json
18:00:47 INFO ar.events.Agenda: fetching https://www.arkleg.state.ar.us/Calendars/Agenda?id=51500430&committee=430&agenda=25699&date=3%2F6%2F2023
18:00:47 INFO scrapelib: GET - 'https://www.arkleg.state.ar.us/Calendars/Agenda?id=51500430&committee=430&agenda=25699&date=3%2F6%2F2023'
18:00:48 INFO openstates: save event 2023-03-06 10:00:00-06:00 PUBLIC HEALTH, WELFARE AND LABOR COMMITTEE - SENATE as event_11a63724-b94e-11ed-a6a9-74e5f9d5f8cc.json
```

**Comment by a-cat-named-snowball** [comment]
I should add - I saw the same missing data on my end when I started trying to debug this. While I was adding logging it started pulling the expected data.

**Comment by chrisyamas** [comment]
@a-cat-named-snowball have you pushed all commits from your local? It's so odd that you are getting that output but I'm getting the same results as before when running the most updated version of the scraper in this PR

**Comment by a-cat-named-snowball** [comment]
@chrisyamas Yes I have, even pulled again just to make sure. Next step for me is to remove repo, reboot, clone repo, check again. If output still differs then I think caching could be the cause of the difference.

**Comment by a-cat-named-snowball** [comment]
Same data on my end after reboot and fresh clone of the repo.

**Comment by a-cat-named-snowball** [comment]
@chrisyamas Sorry the last message was ambiguous. The data still differs, the bills for "JUDICIARY COMMITTEE - SENATE" are being scraped correctly on my end.

**Review by chrisyamas — APPROVED** [review]

**Pull request merged**